internal/history/models.go

```diff
@@ -357,6 +374,29 @@ func (filter *listEvaluationFilter) ExcludedProfileNames() []string {
 	return filter.excludedProfileNames
 }
 
+func (filter *listEvaluationFilter) AddLabel(label string) error {
+	if label == "!*" {
```

Comment:
This looks like the same logic that exists in `LabelsFromFilter`, should it be centralized somewhere so it's not duplicated? https://github.com/mindersec/minder/blob/main/internal/db/domain.go#L49

Reply:
I agree that this logic should be shared, but history list RPC splits the responsibility between the controlplane and the service layer, while `LabelsFromFilter` wraps it in the database layer.

I would update how the `LabelsFromFilter` works by doing the same as done here, but in a separate PR. I can open a task for proper prioritization.